src/ONYXKEYS.ts

```diff
@@ -404,7 +404,7 @@ type OnyxValues = {
     [ONYXKEYS.FORMS.WELCOME_MESSAGE_FORM]: OnyxTypes.Form;
     [ONYXKEYS.FORMS.LEGAL_NAME_FORM]: OnyxTypes.Form;
     [ONYXKEYS.FORMS.WORKSPACE_INVITE_MESSAGE_FORM]: OnyxTypes.Form;
-    [ONYXKEYS.FORMS.DATE_OF_BIRTH_FORM]: OnyxTypes.Form;
+    [ONYXKEYS.FORMS.DATE_OF_BIRTH_FORM]: OnyxTypes.DateOfBirthForm;
```

Comment:
Can you add DATE_OF_BIRTH_FORM_DRAFT key to match changes from FormActions? :)

Reply:
Aren't we going to have merge conflicts if I do that?